### PR TITLE
chore(STONEO11Y-169): limit login to grafana

### PR DESCRIPTION
**Pull request by avi-biton**
Limit login to Grafana UI.
Only users with access to 'appstudio-grafana' namespace will get access to Grafana UI.

### CODE REVIEW

**Comment by avi-biton** [comment]
@gbenhaim @alexeykazakov Please review when you can :)

**Review by amisstea — APPROVED** [review]

**Review by hmariset — APPROVED** [review]
LGTM

File: components/monitoring/grafana/base/grafana-app.yaml [hunk 1]
```diff
@@ -135,7 +135,7 @@ spec:
         - '-http-address='
         - '-email-domain=*'
         - '-upstream=http://localhost:3000'
-        - '-openshift-sar={"resource": "namespaces", "verb": "get"}'
+        - '-openshift-sar={"resource": "namespaces", "resourceName": "appstudio-grafana", "namespace": "appstudio-grafana", "verb": "get"}'
```
Comment (gbenhaim) [inline components/monitoring/grafana/base/grafana-app.yaml:138]: I think that we should also add RBAC permissions so RHTAP developers can view Grafana.
You can add it in the `components/authentication/base/everyone-can-view.yaml` file.

**Review by gbenhaim — CHANGES_REQUESTED** [review]

**Review by gbenhaim — APPROVED** [review]

**Comment by amisstea** [comment]
@avi-biton please clean up the commits and rebase. That may fix the failing e2e tests.

**Review by Kousalya1998 — APPROVED** [review]
/lgtm

**Pull request merged**